jetbrains-core/src/software/aws/toolkits/jetbrains/services/clouddebug/DebuggerSupport.kt

```diff
@@ -121,9 +121,16 @@ abstract class DebuggerSupport {
         val EP_NAME = ExtensionPointName<DebuggerSupport>("aws.toolkit.clouddebug.debuggerSupport")
 
         @JvmStatic
-        fun debuggers(): EnumMap<CloudDebuggingPlatform, DebuggerSupport> = EnumMap(EP_NAME.extensions.map {
-            it.platform to it
-        }.toMap())
+        fun debuggers(): EnumMap<CloudDebuggingPlatform, DebuggerSupport> {
```

Comment:
what the heck is this about?

Reply:
Its the earlier commits from amster getting in... the feature branch was made 2 commits behind master remotely

Reply:
Messed up merge, will be fixed